proto/spacemesh/v1/tx_types.proto

```diff
@@ -53,3 +53,33 @@ message TransactionState {
     }
     TransactionState state = 2;
 }
+
+// TransactionResultsRequest request object for results stream.
+message TransactionResultsRequest {
+    // id is filter by transaction id.
+    bytes id = 1;
+    // address is a filter by account address, it could be principal or any affected address.
+    bytes address = 2;
+    // start streaming from this layer. if 0 - stream will start from genesis.
+    uint32 start = 3;
+    // end streaming at this layer. if 0 - stream till the latest available layer.
+    uint32 end = 4;
+    // watch live data.
+    bool watch = 5;
+}
+
+message TransactionResult {
+    enum Status {
+        SUCCESS = 0;
+        FAILURE = 1;
```

Comment:
it will have status FAILED

malformed transactions in the block means that block is not valid, and node should not apply it

Reply:
Not malformed transactions - transactions that are correctly formed but "ineffective" because principal couldn't pay base gas fee, or nonce was invalid. This is not the same as FAILED.

Reply:
they are FAILED though, but will have different message

Reply:
added INVALID